src/entrypoint/trigger.ts

```diff
@@ -77,20 +72,15 @@ export async function onchainTransactionEventHandler(tx) {
       // transaction has been sent. and this events is trigger before
     }
 
-    await Transaction.updateMany({ hash: tx.id }, { pending: false })
+    await ledger.settlePayment(tx.id)
```

Comment:
it might not be obvious that settlePayment is for onchain. we should make this more precise?

because it take a (transaction) hash but there could be also a settlePayment for (invoice) hash

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/398/commits/f4520630c893dc525f8625a7f04e54901943fe17